### DISCUSSION TITLE
Shaders that require Optifine or iris eg; complementary Reimagined

### DISCUSSION THREAD

**Discussion by Ryan1andonly** [Sinytra/Connector#752] [Q&A]
I have been trying to install shaders such as complementary, but it doesn't work is there any way I can load these shaders with Sinytra connector?

**Comment by ghost** [comment]
https://www.curseforge.com/minecraft/mc-mods/oculus

**Comment by kakashi1464** [comment]
The latest version  of the oculus on CF or MF is broken with the embeddium version higher than 0.2.14.
A version of the oculus that works with the new embeddium is available in the discord server Asek3 or Su5eD.
